ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-provision-chain.yaml

```diff
@@ -31,7 +31,7 @@ chain:
   - name: LOCALZONE_INSTANCE_TYPE
     default: "c5d.2xlarge"
   - name: LOCALZONE_WORKER_SCHEDULABLE
-    default: "yes"
```

Comment:
`byo-subnet` is for phase 1, right?

Reply:
For both 1 and 2.

Reply:
So for now, all the existing install workflow would default `LOCALZONE_WORKER_SCHEDULABLE` to `no`, right?

Reply:
So what is the purpose to keep the env?

Reply:
For future expected case support?

Reply:
> So for now, all the existing install workflow would default LOCALZONE_WORKER_SCHEDULABLE to no, right?

Yes.

> So what is the purpose to keep the env?

This env was added in Phase 0, at the beginning of LZ support, we want more workloads running on LZ nodes, so that we can have more chance to hit errors on LZ, but for customers, we suggest setting LZ nodes `NoSchedualable` (the default behavior) and running the particular workloads on LZ node on-demand. We can remove it from CI, it's not required by any test cases.

Reply:
okay. let us keep it for a while.